### PR TITLE
Fix issue 3393

### PR DESCRIPTION
**Pull request by jkriegshauser**
Fix for #3393

### CODE REVIEW

**Comment by jkriegshauser** [comment]
@khassel @rejas review please?

File: tests/configs/modules/calendar/rrule_until.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/* MagicMirror² Test calendar exdate
```
Comment (rejas) [inline tests/configs/modules/calendar/rrule_until.js:1]: those header are not necessary anymore

**Review by rejas — CHANGES_REQUESTED** [review]

**Comment by rejas** [comment]
sorry for the long delay, lots of life stuff. only a small request change

File: tests/configs/modules/calendar/rrule_until.js (hunk 1 above)
Comment (jkriegshauser) [reply tests/configs/modules/calendar/rrule_until.js]: Alright, removed the header there

**Review by rejas — APPROVED** [review]

**Pull request merged**